src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -591,9 +591,7 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-linux`
SemanticVersion api tests.Error handling.Missing parts: 'NullString': '$null'
```powershell
Expected an exception, with FullyQualifiedErrorId 'FormatException' to be thrown, but the FullyQualifiedErrorId was 'PSArgumentNullException'. from /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1:209 char:19
    +                 { [SemanticVersion]::Parse($version) } | Should -Thro ...
    +                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/SemanticVersion.Tests.ps1: line 209
209:                 { [SemanticVersion]::Parse($version) } | Should -Throw -ErrorId "FormatException"
```
Null Representatives.Comparisons with other null representatives.{ [AutomationNull]::Value } should not be equal to { [DBNull]::Value }
```powershell
Expected $false, but got $true.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 42
42:             $LHS.InvokeReturnAsIs() -eq $RHS.InvokeReturnAsIs() | Should -BeFalse
```
Null Representatives.Comparisons with other null representatives.{ [AutomationNull]::Value } should not be equal to { [NullString]::Value }
```powershell
Expected $false, but got $true.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 42
42:             $LHS.InvokeReturnAsIs() -eq $RHS.InvokeReturnAsIs() | Should -BeFalse
```
Null Representatives.Comparisons with other null representatives.{ [DBNull]::Value } should not be equal to { [NullString]::Value }
```powershell
Expected $false, but got $true.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Language/Scripting/NullRepresentatives.Tests.ps1: line 43
43:             $RHS.InvokeReturnAsIs() -eq $RHS.InvokeReturnAsIs() | Should -BeFalse
```